systers_portal/templates/meetup/base.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% load verbose_name %}
+
+{% block title %}
+  - {{ object.name }}
+{% endblock %}
+
+{% block content %}
+  <div class="mt40"></div>
+  <div class="row">
+    <div class="col-md-12">
+      <h1>{{ object.name }}</h1>
+      <hr/>
+    </div>
+    <div class="col-md-9">
+      <ul class="nav nav-pills">
+        <li> <a href="#">Upcoming</a> </li>
+        <li> <a href="#">Past</a></li>
+        <li> <a href="{% url 'display_meetup_location' slug 'organizers' %}">Organizers</a></li>
```

Comment:
After your comment I understood what you are trying to do.
Besides the drawback explained earlier, it's also a bit confusing to the developer because of the naming - you want to show a separate page/view about organizers using `display_meetup_location` view.